includes/payments/class-payment-stats.php

```diff
@@ -276,7 +276,7 @@ public function get_earnings( $download_id = 0, $start_date = false, $end_date =
 				$statuses = apply_filters( 'edd_payment_stats_post_statuses', $statuses );
 				$statuses = "'" . implode( "', '", $statuses ) . "'";
 
-				$result = $wpdb->get_row( $wpdb->prepare(
+				$results = $wpdb->get_results( $wpdb->prepare(
 					"SELECT edd_oi.tax, edd_oi.total
```

Comment:
You should be able to get the SUM of both of these with something like:
`SELECT SUM(edd_oi.tax) as tax, SUM(edd_oi.total) as total`

That would let you not have to do the `foreach` later on to do the math here. That would be much faster.